src/controller/recordController.js

```diff
@@ -0,0 +1,49 @@
+const Record = require('../models/Record');
+
+const getRecords = async (req, res, next) => {
+  try {
+    const allRecords = await Record.find().sort('title');
+    res.json(allRecords);
+  } catch (err) {
+    next(err);
+  }
+};
+
+// for testing
+const getRecord = async (req, res, next) => {
+  const { id } = req.params; // tbd
+  try {
+    const record = await Record.findById(id);
+    res.json(record);
+  } catch (err) {
+    next(err);
+  }
+};
```

Comment:
Are we using this function? I know it says for testing but if we are not using it we should have it there. It is not good to have code it is not being used at all. You can always save it somewhere, or having in another branch, but it is not good practice to merge it into main.

Reply:
will delete this one